test/cast.c

```diff
@@ -12,4 +26,7 @@ void testmain(void) {
     int a[3];
     *(int *)(a + 2) = 5;
     expect(5, a[2]);
+
```

Comment:
The other test files don't really have white spaces here. I'd wait for a comment from Rui about the white spaces.

Sorry about being picky - Consistency keeps things highly readable.